test.c

```diff
@@ -2267,6 +2267,35 @@ test_refresh_timeout ()
     CU_ASSERT (assert_apteryx_empty ());
 }
 
+
+void
+test_refresh_duplicate ()
+{
+    const char *path = TEST_PATH"/interfaces/eth0/state";
+    const char *path2 = TEST_PATH"/interfaces/eth0/*";
+    char *value = NULL;
+
+    _cb_count = 0;
+    _cb_timeout = TEST_SLEEP_TIMEOUT / 2;
+    CU_ASSERT (apteryx_refresh (path, test_refresh_callback));
+    CU_ASSERT (apteryx_refresh (path2, test_refresh_callback));
+
+    CU_ASSERT ((value = apteryx_get (path)) != NULL);
+    free (value);
+    usleep (TEST_SLEEP_TIMEOUT);
```

Comment:
New paragraph and a comment that this paragraph does a get an only one refresher should be called.